esti/lakectl_test.go

```diff
@@ -884,6 +884,10 @@ func TestLakectlAbuse(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, f.Close())
 
+	const (
+		abuseAmount      = 1000
```

Comment:
```suggestion
		abuseAmount      = 50
```
I am not sure why we need more in order to ensure that `lakectl abuse` continues to work.  If there is some minimal value or guidelines how not to pick too *low* a value, let's add them here as documentation!